### PR TITLE
Exclude 'deps' and '.github' from npm publish

### PR DESCRIPTION
**Pull request by emonddr**
Related to: https://github.com/strongloop/loopback-connector-mssql/issues/222 

Using similar fix applied to loopback-connector-mssql : https://github.com/strongloop/loopback-connector-mssql/pull/226 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-cloudant) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,13 @@
     "posttest": "npm run lint",
     "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require strong-mocha-interfaces --require test/init.js --ui strong-bdd"
   },
+  "files": [
+    "intl",
+    "lib",
+    "docs.json",
+    "index.js",
+    "setup.sh"
```
Comment (jannyHou) [inline package.json]: cloudant contains more setup files I think:
setup.sh
test.setup.sh
test.js
docker.setup.js
docker-compose.yaml

Comment (jannyHou) [reply package.json]: on a second thought, I am not sure do we need to include the setup files when install the connector as a dependency. usually the app should provide a valid db. So probably remove the `setup.sh` file?

Comment (jannyHou) [reply package.json]: back from discussion in 
I see why `setup.sh` is needed in other connectors.
Tip: lb-next/repository-tests has standalone setup script for cloudant, see https://github.com/strongloop/loopback-next/blob/master/acceptance/repository-cloudant/setup.sh, so including/excluding it in `files` doesn't affect the tests. I will honor @emonddr 's decision on it.

Comment (emonddr) [reply package.json]: > back from discussion in https://github.com/strongloop/loopback-connector-mysql/pull/415/files#r391873007
> I see why `setup.sh` is needed in other connectors.
> Tip: lb-next/repository-tests has standalone setup script for cloudant, see https://github.com/strongloop/loopback-next/blob/master/acceptance/repository-cloudant/setup.sh, so including/excluding it in `files` doesn't affect the tests. I will honor @emonddr 's decision on it.

I figure we should be consistent in what we include in "files" section.

**Pull request merged**